exporter/prometheusremotewriteexporter/exporter_test.go

```diff
@@ -13,6 +13,8 @@ import (
 	"testing"
 	"time"
 
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/translator/prometheusremotewrite"
```

Comment:
just move this down to the other collector-contrib import (line 35)

Reply:
thanks that fixed it.